inc/formlist.class.php

```diff
@@ -46,12 +46,11 @@ public static function getTypeName($nb = 0) {
       return _n('Form', 'Forms', $nb, 'formcreator');
    }
 
-   public static function getMenuContent() {
-      $menu = parent::getMenuContent();
-      $menu['title'] = static::getTypeName(Session::getPluralNumber());
-      $menu['page'] = PluginFormcreatorFormList::getSearchURL(false);
-      $menu['icon'] = 'fas fa-edit';
+   public static function canView() {
+      return (PluginFormcreatorForm::countAvailableForm() > 0);
+   }
```

Comment:
In CommonGLPI::getMenuContent() : if  the itemtype is not inherited from CommonDBTM, calls canView() if it exists.

The menu entry must show if there is at least 1 form available for the user. This is done in PluginFormcreatorForm::countAvailableForm.

The menu is created like does CommonGLPI::getMenuContent() (except $menu['links']['search'] which does not matters). Then we can simply drop getMenuContent() here and let the parent implementation do its job